pkg/network/cache/multi.go

```diff
@@ -192,6 +193,13 @@ func (x *multiClient) ReportError(err error) {
 		return
 	}
 
+	x.addrMtx.RLock()
+	group := x.addr
+	x.addrMtx.RUnlock()
+
+	x.opts.Logger.Warn("multi-cache: dropping clients to the node",
```

Comment:
Warn? @532910 won't like it. It's a regular operation thing.

Reply:
nobody except developers will understand this message: what is `multi-cache`?. So, if u wanna make it visible for admins, lets make text clear and then `info`. Otherwise - `debug`

Reply:
I provide my ideas in the commit message why it is an important thing for us but since 2>1 made it INFO

Reply:
it's not about democracy. I didn't got the maintenance flow: does admin do anything when he see this message?

Reply:
> does admin do anything when he see this message?

tries to figure out why some node is dead and objects are going to another node for at least the next 30 seconds

Reply:
imo admin does not care about cache invalidating, only about observable system degradation requiring certain actions for correction. And this is on alerting system plane

Reply:
> only about observable system degradation requiring certain actions for correction

system is degrading, cause one node thinks that another is dead. not for an object, not for a single connection, not for some operation but in general. i do not like this behavior but it was done for failover tests. should be discussed cause it solves something but make it hard in another cases

Reply:
>system is degrading

ok, admin sees this and the message u've added, then do u know

> certain actions for correction

by admin right now? Restart current node/restart problem node/anything else? 

> i do not like this behavior 

i believe, and tryin to realize the problem. Ref me to an existing report about this potential problem or create the new issue pls